### DISCUSSION TITLE
HELLO DEAR DEVELOPER

### DISCUSSION THREAD

**Discussion by kankongsup** [community/community#86579] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

we seem to have a problem with the Pos that When creating a product, when selecting a category, the units do not update according to the units of that category we tried all the way we can but have the same result please fix this
If you don't understand I'm sorry I'm not good with English

**Comment by kmadaminjanova0508** [comment]
It sounds like you're facing an issue in your Point of Sale (POS) system related to product creation and unit selection.Try to do review Code and Database Logic. U can check the code that handles product creation and category selection. Ensure that the logic for updating units based on the selected category is correctly implemented, or verify the database queries associated with these operations to make sure they are retrieving and updating the data as intended. Another thing u can do is check for JavaScript/AJAX issues. If your POS system uses a web interface, ensure that any JavaScript or AJAX calls responsible for updating the units dynamically are functioning correctly.Look for any errors in the browser console that might provide insights into the issue.Test with Sample Data I mean, create a test environment or use a staging version of your POS system to experiment with the product creation process.
Try creating products with various categories to see if the issue is consistent or specific to certain scenarios or verify Data Consistency: Check the consistency of your data. Ensure that categories and units are properly defined in your system and that there are no discrepancies causing the issue :)

**Comment by queenofcorgis** [comment]
Hi @kankongsup ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on your POS system. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
